libnavigator/src/main/java/com/mapbox/navigation/navigator/internal/MapboxNativeNavigatorImpl.kt

```diff
@@ -397,9 +397,6 @@ object MapboxNativeNavigatorImpl : MapboxNativeNavigator {
                             }
                             routeProgressBuilder.bannerInstructions(bannerInstructions)
                         }
-                        ifNonNull(currentStep.bannerInstructions()) {
-                            stepProgressBuilder.guidanceViewURL(getGuidanceViewUrl(it))
```

Comment:
Does moving the url back to banner instructions allow us to omit step calculations?

Reply:
Yes as mentioned above, these were unnecessary. We have all the information in banner instructions.
cc @LukasPaczos